pallets/manta-pay/src/types.rs

```diff
@@ -177,12 +177,15 @@ pub const PROOF_LENGTH: usize = 128;
 /// Transfer Proof Type
 pub type Proof = [u8; PROOF_LENGTH];
 
-///
+/// Asset Identifier Type
 pub type AssetId = [u8; 32];
 
-///
+/// Asset Value Type
 pub type AssetValue = u128;
 
+/// Account Identifier Type
+pub type AccountId = [u8; 32];
```

Comment:
you can import this: `use sp_runtime::AccountId32;`

Reply:
Unfortunately I can't. The point being that now in `manta-rs` (see https://github.com/Manta-Network/manta-rs/pull/324) you need the `AccountId` to sign transactions (before even interacting with the Ledger). The design choice I came up with was to introduce `AccountId` as part of the `Transfer` configuration because it is now necessary to transform a `Transfer` into a `TransferPost`, and then impose compatibility with `Manta` by requiring that `T::AccountId: From<AccountId> + Into<AccountId>` wherever conversions are needed.